### PR TITLE
Dumpling & Cheeseburger: Android 15 BETA release 05-11-2024

### PR DESCRIPTION
**Pull request by Zed9K**
Sorry for the million PR
i have been doing it mostly from my office PC, since im out of town and i still didn't buy a laptop yet!

### CODE REVIEW

File: cheeseburger.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+			"device": "OnePlus 5",
+                        "filename": "crDroidAndroid-15.0-20241105-cheeseburger-v11.0-BETA.zip",
+                        "download": "https://sourceforge.net/projects/crdroid/files/cheeseburger/11.x/crDroidAndroid-15.0-20241105-cheeseburger-v11.0-BETA.zip",
+                        "timestamp": 1730802157,
+                        "md5": "30d88e0a396ef69a6d7dafd787b4126a",
+                        "sha256": "9c866eddcf2e70bdcffc28ba6c17299246a388635ccfbc1d64182f2608620591",
+                        "size": 1093531957,
+                        "version": "11.0",
+                        "buildtype": "BETA",
+		        "forum": "https://xdaforums.com/t/rom-15-0-official-oneplus-5-5t-crdroid-android-11-0-beta-05-nov-2024.4671653/",
```
Comment (gwolf2u) [inline cheeseburger.json:15]: remove telegram link and link to device support page

File: dumpling.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"device": "OnePlus 5T",
+                        "filename": "crDroidAndroid-15.0-20241105-dumpling-v11.0-BETA.zip",
+                        "download": "https://sourceforge.net/projects/crdroid/files/dumpling/11.x/crDroidAndroid-15.0-20241105-dumpling-v11.0-BETA.zip",
+                        "timestamp": 1730791700,
+                        "md5": "b48066f41ed8f236ba26a59a4cf54d44",
+                        "sha256": "baac37c27b245c076b43256b205e2b12cd3f40018a7a20ca2805e7e829930b7f",
+                        "size": 1093611042,
+                        "version": "11.0",
+                        "buildtype": "BETA",
+			"forum": "https://xdaforums.com/t/rom-15-0-official-oneplus-5-5t-crdroid-android-11-0-beta-05-nov-2024.4671653/",
```
Comment (gwolf2u) [inline dumpling.json:15]: same here (remove telegram url and link to support page)

**Review by gwolf2u — CHANGES_REQUESTED** [review]
please fix xda template and upload builds

**Comment by gwolf2u** [comment]
also on your fork, commit changes and squash if you want (squash I can do on merge or PR)
no need to re-open PR

**Comment by Zed9K** [comment]
builds were uploaded to 10.x as a mistake 
and i already deleted them from my machine
should i rebuild ?

+ i will request the PR again in a bit

**Comment by gwolf2u** [comment]
> builds were uploaded to 10.x as a mistake and i already deleted them from my machine should i rebuild ?
> 
>     * i will request the PR again in a bit

moved on SF to proper dir

**Pull request closed without merging**